src/graphics/particle/MagicFlare.cpp

```diff
@@ -283,7 +283,7 @@ long MagicFlareContainer::countWithoutIO() {
 
 	long count = 0;
 	for(size_t i = 0; i < m_magicFlaresMax; i++) {
-		if(g_magicFlares[i].exist && !g_magicFlares[i].hasIO) {
+		if(g_magicFlares[i].exist && !g_magicFlares[i].io) {
```

Comment:
Intentional? .io is cleared when the entity is destructed, .hasIO is not. Not sure if this actually matters.

Reply:
As far as I know it was like this in the original code, but I removed member variable altogether in a later commit.